plugins/importexport/native/filter/NativeXmlArticleFilter.inc.php

```diff
@@ -81,6 +81,9 @@ function handleChildElement($n, $submission) {
 			case 'artwork_file':
 				$this->parseSubmissionFile($n, $submission);
 				break;
+			case 'supplementary_file':
+				$this->parseSubmissionFile($n, $submission);
+				break;
```

Comment:
The two `case` statements for `artwork_file` and `supplementary_file` can be combined into one.